http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/listener/SourceHandler.java

```diff
@@ -57,13 +72,14 @@
     private Map<String, TargetChannel> channelFutureMap = new HashMap<>();
     protected Map<String, GenericObjectPool> targetChannelPool;
     protected ListenerConfiguration listenerConfiguration;
+    private WebSocketServerHandshaker handshaker;
+
 
     public ListenerConfiguration getListenerConfiguration() {
         return listenerConfiguration;
     }
 
-    public SourceHandler(ConnectionManager connectionManager, ListenerConfiguration listenerConfiguration)
-            throws Exception {
```

Comment:
Any reason for the change in the signature ? Why the exception is not throws anymore?